src/components/Api.js

```diff
@@ -0,0 +1,147 @@
+export class Api {
+  constructor({ baseUrl, headers }) {
+    this._url = baseUrl;
+    this._authorization = headers.authorization;
+  }
+
+  getInitialCards() {
+    return fetch(`${this._url}/cards`, {
+      headers: {
+        authorization: this._authorization,
+        "Content-Type": "application/json",
+      },
```

Comment:
According to the task you need to pass full `headers` into the constructor of `Api`. See here https://disk.yandex.ru/i/J93dbJp7eypXKQ . And in the code you need to use only: `headers: this._headers` (not to duplicate the code)